### PR TITLE
BPMSPL-690: Standalone Controller image for OSE

### PR DESCRIPTION
**Pull request by rbarriuso**
This PR depends on the following PRs:

- https://github.com/jboss-container-images/jboss-bpmsuite-7-image/pull/6 
- https://github.com/jboss-openshift/cct_module/pull/30

Also updated the OpenShift tags of the business central and the kie-server.

### CODE REVIEW

File: standalonecontroller/image.yaml [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+          - cct_module.os-eap7-openshift:
+                - user: 185
+                - configure_sh:
+          - cct_module.os-eap7-modules:
+                - user: 185
+                - configure_sh:
+          - cct_module.os-eap7-ping:
+                - user: 185
+                - configure_sh:
+          - cct_module.os-eap70-openshift:
```
Comment (rbarriuso) [inline standalonecontroller/image.yaml:249]: Without this module the container didn't start properly (file `standalone-openshift.xml` not found). I think the `bpmsuite70-executionserver-openshift` image has this problem. Wdyt @goldmann ?

Comment (goldmann) [reply standalonecontroller/image.yaml:249]: You may be right, but I'll defer here to @errantepiphany.

Comment (errantepiphany) [reply standalonecontroller/image.yaml:249]: It looks like the os-eap70-openshift module was added recently @douglaspalmer , but was overlooked into also adding it to the yaml files for both jboss-bpmsuite-7-openshift-image/businesscentral and jboss-bpmsuite-7-openshift-image/executionserver . It should be, for both.

Comment (errantepiphany) [reply standalonecontroller/image.yaml:249]: And it's stuff like this that concerns me about how we do reuse. All the EAP-based images we have have to "replay" all the modules in each of their image.yaml files. So, if a modules is added or removed to EAP, whoever does that work has to remember to do the same thing in all the other EAP-based images. Instead, if we just had those images layer on top of our openshift EAP image, then we wouldn't have to remember to replay all those modules. Just a thought.

Comment (goldmann) [reply standalonecontroller/image.yaml:249]: Noted, this will be resolved in the future versions of Dogen. We already work on this.

Comment (rbarriuso) [reply standalonecontroller/image.yaml:249]: I've created the following PR to add the missing CCT script: https://github.com/jboss-container-images/jboss-bpmsuite-7-openshift-image/pull/7

Comment (rbarriuso) [reply standalonecontroller/image.yaml:249]: @errantepiphany I've also noticed the following error when starting the image of the execution server:

`/opt/eap/bin/launch/bpmsuite-common.sh: line 34: mvn: command not found`

Should I open a JIRA issue about it?

Comment (errantepiphany) [reply standalonecontroller/image.yaml:249]: @rbarriuso Yeah, please open a JIRA. That's weird the maven executable isn't found...

Comment (goldmann) [reply standalonecontroller/image.yaml:249]: The SCL Maven needs this: `. /opt/rh/rh-maven33/enable` in every script that uses the `mvn` command.

Comment (rbarriuso) [reply standalonecontroller/image.yaml:249]: I've submitted a JIRA: https://issues.jboss.org/browse/RHBPMS-4830
I'll work on it following @goldmann suggestion

Comment (rbarriuso) [reply standalonecontroller/image.yaml:249]: PR submitted: https://github.com/jboss-openshift/cct_module/pull/36

**Review by goldmann — APPROVED** [review]

**Comment by rbarriuso** [comment]
@goldmann @errantepiphany Please note that this PR depends on https://github.com/jboss-openshift/cct_module/pull/30 which isn't merged yet

**Pull request merged**